numba/dppl/dppl_offload_dispatcher.py

```diff
@@ -0,0 +1,23 @@
+from numba.core import dispatcher, compiler
+from numba.core.registry import cpu_target, dispatcher_registry
+import numba.dppl_config as dppl_config
+from numba.dppl.compiler import DPPLCompiler
+from numba.core.cpu_options import ParallelOptions
+
+
+class DpplOffloadDispatcher(dispatcher.Dispatcher):
+    targetdescr = cpu_target

```

Comment:
I think we should create our own target that inherits from the CPUTarget. We already have a dppl target but that target is specifically for the GPU code generation. This target will be the target that takes care of any CPU code we might need (like wrappers and calls made to the GPU code)